src/videojs.ads.js

```diff
@@ -1,13 +1,19 @@
-(function(factory){
-    "use strict";
-    if (typeof define === 'function' && define.amd) {
-        define('videojs-contrib-ads', ['videojs'], function(vjs){factory(window, document, vjs);});
-    } else if (typeof exports === 'object' && typeof module === 'object') {
-        factory(window, document, require('video.js'));
-    } else {
-        factory(window, document, videojs);
-    }
-})
+(function (root, factory) {
```

Comment:
even though git thinks it's different, this file is actually our generated file (it has a new header) and the other file is what this file used to be.